text/052-platform-release-cycle.md

```diff
@@ -0,0 +1,174 @@
+```
+                     DO+           O            O   O  O
+coq      --+---(1)---+------(2)----+-----(3)----+---+--+----
+ vX branch/          |             |            |
+           VX+rc tag/              |            |
+                          VX.0 tag/             |
+                                       VX.1 tag/
+
+                                  DI                   DI
+platform  -----------+----(4)------+--------(5)---------+
+           vX branch/              |                    |
+                      VX.0+beta tag/                    |
+                                              VX.0.0 tag/
+Artifacts:
+- D = docker image for Coq (or Coq + the platform)
+- O = OPAM package (O+ means for core-dev, otherwise it is for the main OPAM repo)
+- I = binary installers for Coq platform
+```
+
+## Coq
+
+On time based schedule the RM branches vX.
+
+1. The RM shepherds the few PR which are ready and pins projects tracked by CI
+   (using commit hashes, not necessarily tags).
+   This step should take no more than 2 weeks.
+   - The RM **tags** VX+rc, but produces no binary installers, see the platform release.
+   - The RM writes an **OPAM package**, currently the policy is to upload it to
+     `core-dev`.
+   - Possibly, the RM builds a docker image, so that project maintainers can use it
+     in CI during (4).
+   
+   **No breaking change is allowed from now on, unless a severe problem is found.**
+2. The documentation is updated (eg. the Changes file) and eventual fixes
+   required by the platform are done. Ideally no other change is done.
+   This step should take no more than 2 weeks.
+   - The RM **tags** the .0 release.
+   - The RM writes an **OPAM package** for the main OPAM repository.
+3. In response to a severe bug report Coq devs make an hotfix in master which is
+   backported to vX.
+   - The RM **tags** the point release, possibly as soon
+    as the fix is available and merged.
+   - The RM writes an **OPAM package** for the main OPAM repository.
+
+## Platform
+
+When Coq VX+rc is tagged, the PRM branches vX
+
+4. Starting with the pins made by RM on Coq's CI all packages part of the
+   platform (or its core) are pinned (in accordance with upstreams, which are
+   notified about the ongoing process). Most, if not all, packages are in Coq's
+   CI or the Platform's CI so there should be no surprises.
+   This step should not take more than one month.
+   - The PRM **tags** VX.0+beta tag and publishes the **binary installers**
+   - The PRM may also provide a **docker image** with the entire platform prebuilt
+5. As users pick up the platform and find severe bugs in Coq, the platform picks
+   up point releases of Coq containing hotfixes and eventually extends packages
+   beyond the core set.
+
+### Platform versioning
+
+It is out of the scope of this CEP to chose a versioning schema for the
+platform. In the diagram above we append a .digit to the Coq version (one
+of the current proposals).
```

Comment:
nitpick: this disclaimer looks very OK, but I have one question: assuming "we append a .digit to the Coq version", maybe the bottom rhs tag in the diagram above should be `VX.1.0` instead of `VX.0.0`?

Reply:
That's a technical eye! Kind of fixed.